build.gradle

```diff
@@ -107,12 +111,13 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.4.0'
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '1.4.0'
   compile group: 'uk.gov.hmcts.reform', name: 'http-proxy-spring-boot-autoconfigure', version: '1.1.0'
-  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '0.4.1'
+  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '0.4.2'
 
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-hystrix', version: '1.4.0.RELEASE'
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-hystrix-dashboard', version: '1.4.0.RELEASE'
 
-  compile group: 'com.microsoft.azure', name: 'applicationinsights-web', version: '1.0.10'
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-agent', version: '2.0.0-BETA'
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-web', version: '2.0.0-BETA'
```

Comment:
bleeding edge ;)

Reply:
milestone seemed pretty close: 97%. just dropping java 6 support :D. the config just doesn't work for 1.x series lol